dependency-check-suppressions.xml

```diff
@@ -38,6 +38,7 @@
         <notes>Temporary to unblock pipeline</notes>
         <cve>CVE-2020-36518</cve>
         <cve>CVE-2022-22968</cve>
+        <cve>CVE-2022-25857</cve>
```

Comment:
These vulnerabilities were only suppressed until 2022-06-01. You'll need to either bump the "until" or find a way to resolve the vulnerability.